src/test/prescreen.test.ts

```diff
@@ -0,0 +1,31 @@
+import dayjs from "dayjs";
+import { useUserStateActions } from "../library/user-state-actions-context";
+
+describe("calculations", () => {
+  const birthDate = new Date();
+  birthDate.setFullYear(1957, 4, 23);
+
+  /* async */ function setRetireDate(dateOfBirth: Date, retireAge: number) {
```

Comment:
Nice to have more tests of these kinds of things but with the setRetireDate code seemingly being the same in both cases, maybe this one was just an early version/idea?